src/main/webapp/main-style.css

```diff
@@ -63,6 +63,17 @@ input{
   font-family: 'Open Sans', sans-serif;
 }
 
+
+#submit {
+  background-color:  darkseagreen;
+  border-color: cadetblue;
+  border-style: solid;
+  margin-left: 10px;
+  font-size: 20px;
+  font-style: bold;
+  color: white;
+}
+
```

Comment:
Could you take this out maybe? I think we changed the colors in a different PR to fit the color scheme better

Reply:
Done!